BCStudentSoftwareDevTeam/celts
# Consolidate student search methods

Issue #84 : The logic for Searching students and Volunteers was in two different files but the functions within them had little differences. The main difference was the type of the data they were returning. 

Solution: Combined the two functions into one called searchVolunteersAndStudents in searchVolunteersAndStudents.py

Test:

1. Navigate to /1/1/track_volunteers
2. Click the + sign to get the search modal pop up. 
3. Begin Typing "Zach" and a student named Zach should pop up. 
4. Testing for other places where the student is searched is still in progress since the search function have been used in different branch

![Screen Shot 2021-08-31 at 10 45 21 AM](https://user-images.githubusercontent.com/46546491/131523910-a50f1083-9b33-42ab-b642-8fbbcb33d6f3.png)
